csm/core/services/information.py

```diff
@@ -42,3 +46,77 @@ async def check_compatibility(self, **request_body):
             "reason": reason
         }
         return response
+
+    @Log.trace_method(Log.DEBUG)
+    async def get_topology(self):
+        """
+        Method to fetch the cortx topology
+        :param **request_body: Request body kwargs
+        """
+        plugin_response = self._query_deployment_plugin.get_topology()
+        return plugin_response
+
+    def _filter_from_dict(self, inputdict, keyword):
+        res = inputdict.copy()
+        for item in inputdict:
+            if item != keyword:
+                res.pop(item)
+        return res
+
+    @Log.trace_method(Log.DEBUG)
+    async def get_resources(self, resource):
+        """
+        Method to fetch the cortx topology
+        :param **request_body: Request body kwargs
+        """
+        plugin_response = self._query_deployment_plugin.get_topology()
+        if isinstance(plugin_response['topology'], dict):
+                plugin_response['topology'] = {key:value for key, value in \
```

Comment:
Static strings should be in const

Reply:
Removed all occurrences of static string by adding them in const file